website/blog/2024-06-24-release-1.11.md

```diff
@@ -0,0 +1,249 @@
+- **macOS Rosetta support for Apple Silicon**: Build AMD64 binaries at near-native speeds as ARM64 builds.
+- **Kubernetes improvements**: Check out our new Node and Volume pages for Kubernetes.
+- **Improved UI**: In addition to our light mode, you'll notice updates to our container listing page.
+- **Increased manifest support**: You'll now see images grouped together if built with a manifest.
+
+Podman Desktop 1.11 is now available. [Click here to download it](/downloads)!
+
+<!--truncate-->
+
+---
+
+## Release Details
+
+### Light Mode 💡
+
+We're excited to announce the arrival of our most-requested feature.. light mode! You can now enjoy a brighter interface by toggling the light mode option in the settings section. Please note, this feature is currently experimental as we continue to optimize the rest of Podman Desktop for this new mode.
+
+![Podman-desktop-1-11-hero](img/podman-desktop-release-1.11/light_mode.png)
+
+### macOS Rosetta Support for Apple Silicon
+
+Podman Desktop 1.11 proudly introduces support for macOS Rosetta, facilitating seamless integration with Apple Silicon. This update, part of Podman 5.1, allows users to enable or disable Rosetta support directly through the Podman Settings. With this enhancement, running AMD64 image builds or containers achieves speeds that are nearly identical to those experienced on ARM64 architectures. For more details on the implementation, visit the [GitHub podman pull request](https://github.com/containers/podman/pull/21670).
+
+### Kubernetes Enhancements
+
+In our latest release, we've enhanced Kubernetes functionality by adding node and volume listings. This update aims to improve your Kubernetes management experience within Podman Desktop.
+
+![kubernetes enhancements](img/podman-desktop-release-1.11/nodes.png)
+
+![kubernetes enhancements](img/podman-desktop-release-1.11/pvc.png)
+
+### User Interface Improvements
+
+Take a moment to appreciate the upgraded user interface of Podman Desktop. You'll find that the container listing section, among others, has received significant enhancements, elevating your overall user experience.
+
+### Enhanced Manifest Support
+
+We've made strides in how we handle manifest images. Now, when building a multi-arch image, manifest images are grouped together, simplifying the process and enhancing efficiency.
+
+![manifests](img/podman-desktop-release-1.11/manifest.png)
+
+---
+
+## Other Notable Enhancements
+
+We've added lots of features this release, here are some other highlights:
+
+- feat(PreferenceResource): adding light theme support for some preferences page by @axel7083 in [#7107](https://github.com/containers/podman-desktop/pull/7107)
+- feat: extensions should stay disabled after restart by @deboer-tim in [#7085](https://github.com/containers/podman-desktop/pull/7085)
+- feat: 1.10 release notes by @deboer-tim in [#6984](https://github.com/containers/podman-desktop/pull/6984)
```

Comment:
I usually remove any lines like this that are features, but 'internal', not really interesting to end users, or part of one of the enhancements we've already covered like light mode or kube improvements.